### PR TITLE
Add icons for upload view toggle

### PR DESCRIPTION
**Pull request by jhoward1994**
### What does it do?

Adds icons needed for upload plugin view switch

Ref strapi/strapi#14895

### CODE REVIEW

File: packages/strapi-icons/assets/icons/Grid.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (ronronscelestes) [inline packages/strapi-icons/assets/icons/Grid.svg:1]: I think @gu-stav updated icons to be 24px now 👀

Comment (joshuaellis) [reply packages/strapi-icons/assets/icons/Grid.svg:1]: Should we be defining a width and height here? Wouldn't it be more versatile for the parent to declare that? Or is that gonna break a lot of things?

Comment (gu-stav) [reply packages/strapi-icons/assets/icons/Grid.svg:1]: I think this is fine: all icons do have it at the moment and as long as there is a viewBox attribute I think we are safe.

@ronronscelestes I didn't update the size - only removed padding on some.

**Comment by gu-stav** [comment]
@jhoward1994 We can minify both icons a bit better:

Grid (40% improvement)

```html
<svg width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M.67 5.33h4a.67.67 0 0 0 .66-.66v-4A.67.67 0 0 0 4.67 0h-4A.67.67 0 0 0 0 .67v4a.67.67 0 0 0 .67.66Zm6.66 0h4a.67.67 0 0 0 .67-.66v-4a.67.67 0 0 0-.67-.67h-4a.67.67 0 0 0-.66.67v4a.67.67 0 0 0 .66.66ZM.67 12h4a.67.67 0 0 0 .66-.67v-4a.67.67 0 0 0-.66-.66h-4a.67.67 0 0 0-.67.66v4a.67.67 0 0 0 .67.67Zm6.66 0h4a.67.67 0 0 0 .67-.67v-4a.67.67 0 0 0-.67-.66h-4a.67.67 0 0 0-.66.66v4a.67.67 0 0 0 .66.67Z" fill="#8E8EA9"/></svg>
```

List (50% improvement)
```html
<svg width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="m2.4.1.1-.1h9.4l.1.1v2.2l-.1.1H2.5l-.1-.1V.1ZM0 4.9l.1-.1h9.4l.1.1v2.2l-.1.1H.1L0 7.1V4.9ZM2.5 9.6l-.1.1v2.2l.1.1h9.4l.1-.1V9.7l-.1-.1H2.5Z" fill="#8E8EA9"/></svg>
```

**Review by joshuaellis — APPROVED** [review]

**Pull request merged**